src/App.tsx

```diff
@@ -26,70 +27,135 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
+  if (sortType !== SortType.NONE) {
+    visibleGoods.sort((good1, good2) => {
+      switch (sortType) {
+        case SortType.ALPHABET:
+          return good1.localeCompare(good2);
+
+        case SortType.LENGTH:
+          return Number(good1.length) - Number(good2.length);
+
+        default:
```

Comment:
```suggestion
switch (sortType) {
    case SortType.ALPHABET: 
      visibleGoods.sort((firstGood, secondGood) => (
        firstGood.localeCompare(secondGood);
      )
    
    case .....
```

Reply:
I've renamed these variables, but I don't understand what else you want me to do🤷‍♂️

Reply:
I didn't ask you to rename them)))
Look again at my proposition and find the differences between our codes.
You start switch case inside sort, I make it outside.

Reply:
@4ebupel waiting for your fix

Reply:
@ivan-kadykalo fixed